specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/bms.json

```diff
@@ -3621,6 +3621,54 @@
         }
       }
     },
+    "CloudError": {
```

Comment:
will it be better to ref ErrorResponse? in ./common-types/resource-management/**v2**/types.json#